### PR TITLE
916507 - Enhance Migration Documentation by Adding Information on the PageChanged Event

### PR DESCRIPTION
**Pull request by Bharathi4332**
Updated the migration.md file

### CODE REVIEW

File: MAUI/PDF-Viewer/Migration.md [hunk 1]
```diff
@@ -318,25 +318,28 @@ To make migration from [Xamarin SfPdfViewer](https://www.syncfusion.com/xamarin-
...
+pdfViewer.PropertyChanged += PdfViewer_PropertyChanged;
 
-    }
+private void PdfViewer_PropertyChanged(object? sender, PropertyChangedEventArgs e)
+{
+   // Check if the changed property is PageNumber
+   if (e.PropertyName == nameof(SfPdfViewer.PageNumber) && sender is SfPdfViewer)
+   {
+      // Get the current page number
+      int currentPageNumber = ((SfPdfViewer)sender).PageNumber;
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Migration.md:341]: Don't use direct casting as it will cause exception. use **is** type of casting.

https://github.com/essential-studio/maui-pdfviewer/blob/development/maui/src/PDFViewer/Controls/SfPdfViewer.cs#L818

File: MAUI/PDF-Viewer/Migration.md [hunk 2]
```diff
@@ -318,25 +318,28 @@ To make migration from [Xamarin SfPdfViewer](https://www.syncfusion.com/xamarin-
 </tr>
 <tr>
 <td>
-<div>{{'[PageChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_PageChanged)'| markdownify }},</div>
+<div>{{'[PageChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfPdfViewer.XForms.SfPdfViewer.html#Syncfusion_SfPdfViewer_XForms_SfPdfViewer_PageChanged)'| markdownify }}</div>
```
Comment (Deepak1799) [inline MAUI/PDF-Viewer/Migration.md:321]: Is this change necessary?

**Review by Deepak1799 — CHANGES_REQUESTED** [review]

File: MAUI/PDF-Viewer/Migration.md (hunk 1 above)
Comment (Bharathi4332) [reply MAUI/PDF-Viewer/Migration.md]: addressed the feedback by changing the code to is type of casting

**Review by Deepak1799 — APPROVED** [review]
